### PR TITLE
fix(client): createManyAndReturn types

### PR DESCRIPTION
**Pull request by Jolg42**
Related to https://github.com/prisma/prisma/pull/24064

Closes https://github.com/prisma/team-orm/issues/1142

Co-authored-by: Serhii Tatarintsev <tatarintsev@prisma.io>

### CODE REVIEW

**Review by Druue — APPROVED** [review]
✨

File: packages/client/src/generation/TSClient/Model.ts [hunk 1]
```diff
@@ -41,13 +43,16 @@ import { getModelActions } from './utils/getModelActions'
...
+  protected createManyAndReturnType: undefined | DMMF.OutputType
   protected mapping?: DMMF.ModelMapping
   private dmmf: DMMFHelper
   private genericsInfo: GenericArgsInfo
   constructor(protected readonly model: DMMF.Model, protected readonly context: GenerateContext) {
     this.dmmf = context.dmmf
     this.genericsInfo = context.genericArgsInfo
     this.type = this.context.dmmf.outputTypeMap.model[model.name]
+
+    this.createManyAndReturnType = this.context.dmmf.outputTypeMap.model[`CreateMany${model.name}AndReturnOutputType`]
```
Comment (SevInf) [inline packages/client/src/generation/TSClient/Model.ts:55]: Afterthought could we pull that dmmf type name creation into it's own `getCreateManyAndReturnOutputType` helper?

Comment (Jolg42) [reply packages/client/src/generation/TSClient/Model.ts:55]: I think that's a very good idea

Comment (Jolg42) [reply packages/client/src/generation/TSClient/Model.ts]: Done!

**Review by SevInf — APPROVED** [review]

**Pull request merged**